src/components/Navigation/Navigation.js

```diff
@@ -14,7 +14,7 @@ function Navigation() {
       </Link>
       <ul className="flex-space_between-center" id="nav-icons">
         <li className="mx-2">
-          <Link to={"/"} href="/Pages/login.html">
+          <Link to={"/"}>
             <button className="btn-primary-confirm">Login</button>
           </Link>
```

Comment:
dont use button inside link, either use link or button + useNavigate to navigate, depending upon the use case